openedx_webhooks/tasks/github.py

```diff
@@ -53,46 +58,43 @@ def pull_request_opened(pull_request, ignore_internal=True, check_contractor=Tru
     is_beta = is_beta_tester_pull_request(pr)
 
     if is_internal_pr and not has_cl and is_beta:
-        logger.info(
-            "Adding cover letter template to PR #{num} against {repo}".format(
-                repo=repo, num=num,
-            ),
-        )
+        msg = "Adding cover letter template to PR #{num} against {repo}".format(repo=repo, num=num)
+        log_info(self.request, msg)
         coverletter = github_internal_cover_letter(pr),
 
         if coverletter is not None:
             comment = {
                 "body": coverletter
             }
-            url = "/repos/{repo}/issues/{num}/comments".format(
-                repo=repo, num=num,
-            )
+            url = "/repos/{repo}/issues/{num}/comments".format(repo=repo, num=num)
 
             comment_resp = github.post(url, json=comment)
+            log_request_response(self.request, comment_resp)
```

Comment:
I think so. We would have two log statements, one request to GitHub and one response from GitHub. In the response, we're logging status code and reason, so we should see `201 Created` if all is well, or an error code if something went wrong.